docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,144 @@
+
+>**NOTE:** If the migration process skipped the Api resources due to the invalid status or a blacklisted label, migration is not possible.
+
+Here are the steps you need to follow to ensure your services are properly migrated:
+1. [Verify the migration outcome](#verify-automatic-migration). 
+2. If you can still see any Api CRs in use, use the [manual migration](#manual-migration) guide to migrate them.
+
+## Verify the automatic migration
+
+Follow these steps to verify if all Api CRs were migrated to APIRule CRs.
+
+1. Once the Kyma upgrade finishes, list the existing Api CRs:
+
+    ```shell script
+      kubectl get apis --all-namespaces
+    ```
+
+2. As Api CRs are removed after a successful migration, the list shows all resources skipped by the migration job. Fetch the logs from the job's Pod to find out the reasons: 
+
+    ```shell script
+    kubectl logs api-gateway-api-migrator-job -n kyma-system
+    ```
+
+## Manual migration
+
+This guide shows how you can manually migrate Api CRs to APIRule CRs.
+
+>**NOTE:** Before you start the manual migration process, see the [Api CR](/components/api-gateway/#custom-resource-api-sample-custom-resource) and [APIRule CR](/components/api-gateway-v2#custom-resource-api-rule) documents for the custom resource detailed definition.
+
+Follow these steps:
+
+1. Fetch the Api CRs you want to migrate:
+
+    ```shell script
+    kubectl get api {NAME} -n {NAMESPACE} -o yaml
+    ```
+
+2. Create an APIRule CR based on the Api CR's specification.
+
+>**NOTE:** Do not copy the [**status**](components/api-gateway-v2#custom-resource-api-rule-additional-information) parameter from the original Api CR.
+
+| Parameter | Action  |
+|-----------|---------|
+| **metadata.name**| Specify the name of the API Rule.| 
+| **spec.gateway**| Replace the default value (`kyma-gateway.kyma-system.svc.cluster.local`) with Istio Gateway used to expose services.|
+| **spec.service.name**| Copy the value of the corresponding parameter of the Api CR.|
+| **spec.service.port**| Copy the value of the corresponding parameter of the Api CR.|
+| **spec.service.host**| Set the value for **service.host** parameter to any temporary value which includes the domain. For example, if the value for the **hostname** parameter of the Api CR was set to `sample-service.kyma.local`, change it to `temp-sample-service.kyma.local` so that differs from the original value. Make sure other services on your cluster do not use this hostname. |
+| **spec.rules**| The **spec.rules** parameter allows you to provide authentication by defining a list of paths along with the authentication methods. The way of handling authentication differs for Api CR and APIRule CR. Consider the section below as a point of reference when defining this part of specification.|
+
+Use the **spec.rules** parameter to specify the authentication type per path and use a regular expression to match every path, a subset of possible paths, or a single one.  
+
+See the example:
```

Comment:
If we need the Api CR example for comparison, we can rewrite it to:

See the example of an Api CR:
```
authentication:
- type: JWT
  jwt:
    issuer: https://dex.kyma.local
    jwksUri: http://dex-service.kyma-system.svc.cluster.local:5556/keys
    triggerRule:
      excludedPaths:
      - exact: /exact/path/to/resource.jpg
      - exact: /no/auth/needed/resource.html
- type: JWT
  jwt:
    issuer: https://auth.kyma.local
    jwksUri: http://auth-service.kyma-system.svc.cluster.local:5556/keys
    triggerRule:
      excludedPaths:
      - prefix: /pref/
      - exact: /no/auth/needed/resource.html
```
In this configuration:

To access the /exact/path/to/resource.jpg path, you need a token issued by https://auth.kyma.local.
To access any path starting with /pref/ you need a token issued by https://dex.kyma.local.
To access the /no/auth/needed/resource.html you don't need any token because the path is excluded for both configurations.
To access all other paths you need a token from one of the issuers.

The APIRule corresponding to this example would look as follows:

.... and the rest follows.

Reply:
I think that it would be better to have the example.